docs/driver.md

```diff
@@ -12,14 +12,15 @@ The command used to launch the IDE. By default, the driver uses the `idea` comma
 By default, the driver waits `30 seconds` for the IDE to connect the probe. After exceeding this time, the whole test fails.  
 #### Automatic checks
 ```
-driver.checks {
+driver.check {
     errors = false
     freezes = false
+    ignoreErrorsWithMessageContaining = []
```

Comment:
Hmm I see both `snakeCase` and `kebab-case` conventions for HOCON in the wild, but I'd say kebab-case is more prevalent... consider reconfiguring pureconfig library if it expects `snakeCase` (although my impression was that `kebab-case` was the pureconfig's default 🤔)

Reply:
now I change configuration that `kebab-case ` need to be in configuration file and `snakeCase` (as it was configured before) in scala case class